bin/stringfuzzx

```diff
@@ -160,6 +195,9 @@ def main():
         print(e, file=sys.stderr)
         return 1
 
+    # filter out suppressed expressions
+    ast = list(filter(is_not_suppressed, ast))
```

Comment:
To preserve semantics, we should skip this step if we're doing the `nop` transformer.